src/sections/pools/table/PoolsTable.utils.tsx

```diff
@@ -26,9 +26,9 @@ import { GlobalFarmRowMulti } from "sections/pools/farms/components/globalFarm/G
 import { Button, ButtonTransparent } from "components/Button/Button"
 import ChevronRightIcon from "assets/icons/ChevronRight.svg?react"
 import ManageIcon from "assets/icons/IconEdit.svg?react"
-import { BN_0, BN_NAN } from "utils/constants"
+import { BN_0, BN_NAN, GETH_ERC20_ASSET_ID } from "utils/constants"
 import Skeleton from "react-loading-skeleton"
-import BN from "bignumber.js"
+import BN, { BigNumber } from "bignumber.js"
```

Comment:
minor thing: you can use BN for all cases you use BigNumber, so no need to import a new class